docs/internal/dev_guidelines.rst

```diff
@@ -80,6 +80,25 @@ These apply to C++ source code, Perl scripts, MPC files, and any other plaintext
   I don't think it makes sense to strictly enforce an 80-column limit, but overly long lines are harder to read.
   Try to keep lines to roughly 80 characters.
 
+The :ref:`lint script <dev_guidelines-lint-script>` will help check for most of these in a PR.
+
+There is also a :ghfile:`.editorconfig` file that allows contributors to follow most of these rules automatically.
+`EditorConfig <https://editorconfig.org/>`__ support is `built-in to some editors (including Visual Studio) <https://editorconfig.org/#pre-installed>`__ with `plugins available for others <https://editorconfig.org/#download>`__.
+
+.. _dev_guidelines-lint-script:
+
+***********
+Lint Script
+***********
+
+The :ghfile:`lint script <tools/scripts/lint.pl>` is a Perl script that is run on every PR.
+It checks for some mistakes both in coding and style.
+See the default checks by running ``./tools/scripts/lint.pl --list``.
```

Comment:
In general I'm not in favor of using `./` prefixes for commands.
We should not assume any particular kind of shell / command interpreter is being used.